WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteFragment.kt

```diff
@@ -519,10 +526,11 @@ class MySiteFragment : Fragment(R.layout.my_site_fragment),
 
     private fun MySiteFragmentBinding.loadData(cardAndItems: List<MySiteCardAndItem>) {
         recyclerView.setVisible(true)
+        actionableEmptyView.setVisible(false)
+        hideRefreshIndicatorIfNeeded()
```

Comment:
Suggestion (🔍): Consider moving this up a level. Maybe this doesn't belong to the `loadData(...)` method because it is not loading any data, wdyt?
```
            when (val state = uiModel.state) {
                is State.SiteSelected -> {
                    hideRefreshIndicatorIfNeeded()
                    loadData(state.cardAndItems)
                }
                is State.NoSites -> loadEmptyView(state.shouldShowImage)
            }
```

Reply:
Moving it out into setupObservers

Reply:
in a147c0e